src/components/search-result-list/search-result-list-item/search-result-list-item.tsx

```diff
@@ -56,8 +57,7 @@ const SearchResultListItem: React.FC<SearchResultListItemProps> = ({
     t
   );
   const manifestationPid = getManifestationPid(manifestations);
-  const firstInSeries = series?.[0];
-  const { title: seriesTitle, numberInSeries } = firstInSeries || {};
+  const firstItemInSeries = hasNumberInSeries(series)[0];
```

Comment:
Consider using `shift()` instead of indexed access.

This should ensure that Typescript sees the return value as `Work["series"] | undefined`. You add a proper check for this in L122 anyway so we should be good but using `shift()` will let TypeScript help you remember this.